capstone/src/main/java/com/google/sps/data/ReadData.java

```diff
@@ -85,75 +103,63 @@ public void readFile(FileReader file, String zipCode) {
 
         final int START_INDEX = 11;
 
-        int indexOfLong = (values.length - EXPECTED_LENGTH) + START_INDEX + 1;
+        int indexOfInt = (values.length - EXPECTED_LENGTH) + START_INDEX + 1;
 
-        for (int i = START_INDEX; i < indexOfLong; i++) {
+        for (int i = START_INDEX; i < indexOfInt; i++) {
           text += values[i];
         }
 
         float score = 0;
 
-        if (text.length() > 0) {
+        if (sentimentService != null && text.length() > 0) {
           score = sentimentService.getSentiment(text);
         }
 
-        long responseTimeOne = Long.parseLong(values[indexOfLong++]);
-        long responseTimeTwo = Long.parseLong(values[indexOfLong++]);
-        long responseTimeThree = Long.parseLong(values[indexOfLong]);
+        int responseTimeOne = (int) Math.round(Double.parseDouble(values[indexOfInt++]));
+        int responseTimeTwo = (int) Math.round(Double.parseDouble(values[indexOfInt++]));
+        int responseTimeThree = (int) Math.round(Double.parseDouble(values[indexOfInt]));
 
         Entity entity = new Entity("Response", id);
-        Entity inStore; // Entity that may or may not exist in Datastore
 
-        try {
-          inStore = datastoreService.get(entity.getKey());
-        } catch (EntityNotFoundException e) {
-          inStore = null;
+        entity.setProperty("zipCode", zipCode);
+        entity.setProperty("id", id);
+        entity.setProperty("date", date);
+        entity.setProperty("completionStatus", completionStatus);
```

Comment:
In the future I think it would be really cool if we could create a file just with all the property name constants in it and then all draw from there to help make our code consistent and guard against typos 😄

Reply:
I agree, having this many properties lead to a lot of issues when refactoring property names